clients/linux/src/app.rs

```diff
@@ -43,11 +43,13 @@ use crate::settings::Settings;
 use crate::util;
 use crate::{closure, progerr, tree_iter_value, uerr, uerr_dialog};
 
+use gdk::{Cursor, WindowExt};
 use glib::uri_unescape_string;
 use lockbook_core::model::client_conversion::ClientFileMetadata;
 use lockbook_core::service::import_export_service::ImportExportFileInfo;
 use std::path::PathBuf;
 
+#[macro_export]
```

Comment:
Does this macro export need to still be here? Or the one for `spawn` below?

Reply:
It seems it doesn't, will remove it.